src/algebra/covariant_and_contravariant.lean

```diff
@@ -28,7 +28,7 @@ relation (typically `(≤)` or `(<)`), these are the only two typeclasses that I
 The general approach is to formulate the lemma that you are interested in and prove it, with the
 `ordered_[...]` typeclass of your liking.  After that, you convert the single typeclass,
 say `[ordered_cancel_monoid M]`, into three typeclasses, e.g.
-`[left_cancel_semigroup M] [partial_order M] [covariant_class M M (function.swap (*)) (≤)]`
+`[left_cancel_semigroup M] [partial_order M] [covariant_class M M (swap (*)) (≤)]`
```

Comment:
Unfortunately I think we need to keep `function.swap` in docstrings, otherwise the links will go away.

Reply:
Good point. Fixed!